includes/class-wsuwp-admin.php

```diff
@@ -645,6 +717,82 @@ public function all_plugins( $plugins ) {
 			'wp-event-calendar/wp-event-calendar.php' => array(
 				'stage.events.wsu.edu/',
 			),
+			'CAHNRSWSUWP-Plugin-Pagebuilder/plugin.php' => array(
+				'stage.web.wsu.edu/cahnrs/',
+				'stage.web.wsu.edu/cahnrs-academics/',
+				'stage.web.wsu.edu/cahnrs-alumni/',
+				'stage.web.wsu.edu/cahnrs-communications/',
+				'stage.web.wsu.edu/cahnrs-wolf-livestock-research/',
+				'stage.web.wsu.edu/cahnrs-cosmiccrisp/',
+				'stage.web.wsu.edu/cahnrs-research/',
+				'stage.web.wsu.edu/cahnrs-tidal/',
+				'stage.web.wsu.edu/cahnrs-honors/',
+				'stage.web.wsu.edu/cahnrs-ciser/',
+				'stage.web.wsu.edu/cahnrs-ctll/',
+				'stage.web.wsu.edu/cahnrs-communications-stage/',
+				'stage.web.wsu.edu/cahnrs-reconnect/',
+				'stage.web.wsu.edu/cahnrs-about/',
+				'stage.web.wsu.edu/cahnrs-pioneeringideas2017/',
+				'stage.web.wsu.edu/cahnrs-ignite/',
+				'stage.web.wsu.edu/cahnrs-tci/',
+				'stage.web.wsu.edu/cahnrs-accessibility/',
+				'cahnrs.wsu.edu/',
+				'cahnrs.wsu.edu/academics/',
+				'cahnrs.wsu.edu/alumni/',
+				'cahnrs.wsu.edu/communications/',
+				'cahnrs.wsu.edu/wolf-livestock-research/',
+				'cahnrs.wsu.edu/cosmiccrisp/',
+				'cahnrs.wsu.edu/research/',
+				'cahnrs.wsu.edu/tidal/',
+				'cahnrs.wsu.edu/honors/',
+				'cahnrs.wsu.edu/ciser/',
+				'cahnrs.wsu.edu/ctll/',
+				'cahnrs.wsu.edu/communications-stage/',
+				'cahnrs.wsu.edu/reconnect/',
+				'cahnrs.wsu.edu/about/',
+				'cahnrs.wsu.edu/pioneeringideas2017/',
+				'cahnrs.wsu.edu/ignite/',
+				'cahnrs.wsu.edu/tci/',
+				'cahnrs.wsu.edu/accessibility/',
+			),
+			'CAHNRSWSUWP-Plugin-Pagebuilder/plugin.php' => array(
```

Comment:
This looks like a duplicate of the array above :)

Reply:
And so it is 🤦‍♂️